AndroidSopt/app/src/main/java/com/example/androidsopt/view/Profile/ProfileFragment.kt

```diff
@@ -59,8 +60,13 @@ class ProfileFragment : Fragment() {
 
     }
 
+    private fun initSettings() {
 
-
+        binding.ibSettings.setOnClickListener {
+            val intent = Intent(this@ProfileFragment.context, SettingActivity::class.java)
+            startActivity(intent)
+        }
```

Comment:
```suggestion
        binding.ibSettings.setOnClickListener {
            val intent = Intent(requireContext(), SettingActivity::class.java)
            startActivity(intent)
        }
```

Fragment에서 Context 사용할 때는 ``requireContext()`` 사용하는 것이 좋습니다(nullable 하지 않음)